README.md

```diff
@@ -13,12 +13,16 @@ First, you need to have JDK 8, Apache Cassandra, and Maven installed in order to
 the project.
 
 Once you have made modifications, here's how you build the project:
- 1. Run `mvn install` to install all of the dependencies (You can also make your
+ 1. Rename `application.example.properties` to `application.properties` and fill in the credentials. This can be found at
+    `src/main/resources`.
+ 2. Open `cqlsh` and run `CREATE KEYSPACE stacksurveyor WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 };` to create
+    a keyspace for the API.
+ 3. Run `mvn install` to install all of the dependencies (You can also make your
     IDE do it).
```

Comment:
is it possible to put all the readme content to limit at 80 characters please

Reply:
I'm sorry, I didn't understand it

Reply:
the current lines are truncated at some other length. is it possible to increase the characters (in a line) limit to 80 ? this is what most projects use :)

Reply:
Alright, I will make a commit on it.

Reply:
okay thanks